ipalib/cli.py

```diff
@@ -200,7 +200,7 @@ def print_plain(self, string):
         """
         Print exactly like ``print`` statement would.
         """
-        print(unicode(string))
+        logger.info_cmd(unicode(string))
```

Comment:
``ipalib.run`` implements the ``ipa`` command line tool. You have to keep all ``print()`` calls.

Reply:
Fixed